pkg/server/controller.go

```diff
@@ -256,6 +260,17 @@ func (ctrl *Controller) serverMux() (http.Handler, error) {
 		ctrl.drainMiddleware,
 		ctrl.authMiddleware(nil))
 
+	if ctrl.adminController != nil {
+		adminRouter := apiRouter.PathPrefix("/admin/").Subrouter()
+		adminRouter.Use(
+			api.AuthMiddleware(nil, ctrl.authService, ctrl.httpUtils),
+			authz.NewAuthorizer(ctrl.log, httputils.NewDefaultHelper(ctrl.log)).RequireOneOf(
+				authz.Role(model.AdminRole),
+			))
+		adminRouter.HandleFunc("/apps", ctrl.adminController.HandleGetApps).Methods("GET")
+		adminRouter.HandleFunc("/apps", ctrl.adminController.HandleDeleteApp).Methods("DELETE")
+	}
+
```

Comment:
It looks like `/apps` depend on the `svc.config.EnableExperimentalAdmin` option. I'm not sure we want this dependency.